github/apps.go

```diff
@@ -194,10 +211,10 @@ func (s *AppsService) ListUserInstallations(ctx context.Context, opt *ListOption
 // CreateInstallationToken creates a new installation token.
 //
 // GitHub API docs: https://developer.github.com/v3/apps/#create-a-new-installation-token
-func (s *AppsService) CreateInstallationToken(ctx context.Context, id int64) (*InstallationToken, *Response, error) {
+func (s *AppsService) CreateInstallationToken(ctx context.Context, id int64, body ...interface{}) (*InstallationToken, *Response, error) {
```

Comment:
Please let's not use `...interface{}` but keep this repo strictly typed.
There are many other examples where we pass in a pointer to an optional struct and we usually call it `opt`, as in:
`opt *InstallationTokenOptions`

Reply:
Done.